Source/JavaScriptCore/inspector/protocol/CSS.json

```diff
@@ -462,6 +477,14 @@
                 { "name": "fontFamilyNames", "type": "array", "items": { "type": "string" }, "description": "Supported system font families." }
             ]
         },
+        {
+            "name": "getCustomFonts",
+            "description": "Returns all loaded fonts (system and custom).",
```

Comment:
if this also includes system fonts, why is it called `getCustomFonts`?  wouldn't something like `getActiveFonts` or `getKnownFonts` be better?

also if this includes system fonts do we still need `getSupportedSystemFontFamilyNames`?